{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"remoteServiceInternalError"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":1,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/galenit_modulessdk\/autherror\/remoteserviceinternalerror"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.galenit.GalenIT-ModulesSDK\/documentation\/GalenIT_ModulesSDK\/AuthError\/remoteServiceInternalError","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"No overview available."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"remoteServiceInternalError"}],"title":"AuthError.remoteServiceInternalError","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:18GalenIT_ModulesSDK9AuthErrorO021remoteServiceInternalF0yA2CmF","modules":[{"name":"GalenIT_ModulesSDK"}]},"hierarchy":{"paths":[["doc:\/\/com.galenit.GalenIT-ModulesSDK\/documentation\/GalenIT_ModulesSDK","doc:\/\/com.galenit.GalenIT-ModulesSDK\/documentation\/GalenIT_ModulesSDK\/AuthError"]]},"references":{"doc://com.galenit.GalenIT-ModulesSDK/documentation/GalenIT_ModulesSDK/AuthError/remoteServiceInternalError":{"role":"symbol","title":"AuthError.remoteServiceInternalError","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"remoteServiceInternalError"}],"abstract":[],"identifier":"doc:\/\/com.galenit.GalenIT-ModulesSDK\/documentation\/GalenIT_ModulesSDK\/AuthError\/remoteServiceInternalError","kind":"symbol","type":"topic","url":"\/documentation\/galenit_modulessdk\/autherror\/remoteserviceinternalerror"},"doc://com.galenit.GalenIT-ModulesSDK/documentation/GalenIT_ModulesSDK/AuthError":{"role":"symbol","title":"AuthError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"AuthError"}],"abstract":[],"identifier":"doc:\/\/com.galenit.GalenIT-ModulesSDK\/documentation\/GalenIT_ModulesSDK\/AuthError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"AuthError"}],"url":"\/documentation\/galenit_modulessdk\/autherror"},"doc://com.galenit.GalenIT-ModulesSDK/documentation/GalenIT_ModulesSDK":{"role":"collection","title":"GalenIT_ModulesSDK","abstract":[],"identifier":"doc:\/\/com.galenit.GalenIT-ModulesSDK\/documentation\/GalenIT_ModulesSDK","kind":"symbol","type":"topic","url":"\/documentation\/galenit_modulessdk"}}}